web/js/functions.js

```diff
@@ -990,19 +990,17 @@ function handleACL() {
 					});
 					playlist.sortable({
 						start: function (event, ui) {
-							PLAYLIST_DRAGFROM = ui.item.index();
-							PLAYLIST_DRAGSANITY = ui.item.data('plobject').videoid;
+							ui.item.parent().data('drag', {
```

Comment:
Couldn't this just use a local variable? Is there a benefit to storing the stuff in jQuery data?